docs/src/config/ini-config.adoc

```diff
@@ -237,15 +237,15 @@ is .inc.  Do not use a file extension of .ini for included files.
 
 Different user interface programs use different options, and not every
 option is supported by every user interface. There are several interfaces,
-like Axis, Gmoccapy, Touchy, QtVCP's QtDragon and Gscreen.
+like AXIS, GMOCCAPY, Touchy, QtVCP's QtDragon and Gscreen.
 Axis is an interface for use with normal
-computer and monitor, Touchy is for use with touch screens. Gmoccapy
+computer and monitor, Touchy is for use with touch screens. GMOCCAPY
 can be used both ways and offers also many connections for hardware controls.
 Descriptions of the interfaces are in the Interfaces section of the
 User Manual.
 
 * 'DISPLAY = axis' - The name of the user interface to use. Valid options
-  may include: 'axis', 'touchy', 'gmoccapy', 'gscreen', 'tklinuxcnc', 'qtvcp'
+  may include: 'AXIS', 'Touchy', 'GMOCCAPY', 'GScreen', 'TKLinuxCNC', 'QtVCP'
```

Comment:
"GScreen"

if I remember correctly, this isn't the preferred wording by @c-morley. But maybe i'm wrong.

Reply:
I think they should be lowercase as they refer to the "executable".

Possibly qtdragon, qtplasmac, and mdro could be added.

Reply:
They need to be lower case as Phill said. 
Not sure about details of mdro, but qtdragon and qtplasmac must be applied with a preceding 'qtvcp'

Reply:
Rebased to master and done.